src/web/services/link.service.ts

```diff
@@ -100,7 +99,7 @@ export class LinkService {
 
     this.filterEmptyParams(params);
     const encodedParams: string = this.navigationService.encodeParams(params);
-    return `${frontendUrl}${this.URI_PREFIX}${this.INSTRUCTOR_STUDENT_RECORDS_PAGE}${encodedParams}`;
+    return `${this.URI_PREFIX}${this.INSTRUCTOR_STUDENT_PROFILE_PAGE}${encodedParams}`;
```

Comment:
@samuelfangjw So what's the purpose of adding `${frontendUrl}` before `${this.URI_PREFIX}` when generating some links?

Reply:
The original post of the issue being linked here should give you the answer

Reply:
Oh I see.